conn.go

```diff
@@ -1500,6 +1500,38 @@ func QuoteIdentifier(name string) string {
 	return `"` + strings.Replace(name, `"`, `""`, -1) + `"`
 }
 
+// QuoteLiteral quotes a 'literal' (e.g. a parameter, often used to pass literal
+// to DDL and other statements that do not accept parameters) to be used as part
+// of an SQL statement.  For example:
+//
+//    exp_date := pq.QuoteLiteral("2023-01-05 15:00:00Z")
+//    err := db.Exec(fmt.Sprintf("CREATE ROLE my_user VALID UNTIL %s", exp_date))
+//
+// Any single quotes in name will be escaped. Any string that contains a backslash
+// (i.e. "\")
```

Comment:
Can you fix this sentence? It doesn't have a verb or period.

Reply:
@mjibson Thanks for the feedback! This has been addressed:

https://github.com/lib/pq/pull/852/files#diff-a2ddca1600f34abb91244424f8431c2cR1511

I also rebased against master.